example/src/main/java/com/stripe/example/activity/LaunchPaymentSheetCompleteActivity.kt

```diff
@@ -15,7 +15,7 @@ internal class LaunchPaymentSheetCompleteActivity : BasePaymentSheetActivity() {
         setContentView(viewBinding.root)
 
         viewModel.inProgress.observe(this) {
-            viewBinding.progressBar.visibility = if (it) View.VISIBLE else View.INVISIBLE
+            viewBinding.progressBar.isVisible = it
```

Comment:
You can change this to `.isInvisible = !it` to keep existing behavior. The way you have it will set to `GONE` when `false`.